store/commitment/store.go

```diff
@@ -210,7 +215,7 @@ func (c *CommitStore) Commit(version uint64) (*proof.CommitInfo, error) {
 			}
 		}
 		storeInfos = append(storeInfos, proof.StoreInfo{
-			Name:     storeKey,
+			Name:     internal.UnsafeStrToBytes(storeKey),
```

Comment:
also here it's not safe to return this, we should create a copy doing : ``[]byte(storeKey)``